src/DataItems/Property.php

```
@@ -403,7 +401,7 @@ public function findPropertyValueType(): string {
 			return $this->propertyValueType;
 		}
 
-		$diWikiPage = new DIWikiPage( $this->getKey(), SMW_NS_PROPERTY, $this->interwiki );
+		$diWikiPage = new WikiPage( $this->getKey(), SMW_NS_PROPERTY, $this->interwiki );
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$diWikiPage'. (undefined)

(UnusedLocalVariable)